container_service_extension/sample_generator.py

```diff
@@ -72,6 +72,12 @@
     }
 }
 
+SAMPLE_MQTT_CONFIG = {
+    'mqtt': {
```

Comment:
interesting, is there bare minimum config necessary for MQTT? 
So unlike AMQP, we don't need to configure sample host/port/username/password etc?
I am unfamiliar with MQTT but this seems a little confusing to me...
Please double check, and in case its actually not necessary, please add a comment describing min sample config for MQTT in here.
That will educate people like me :)

Reply:
For mqtt, we register the extension directly using the cloudapi and api clients, so we don't need information such as the hose, port, etc. We need the verify_ssl flag to see whether or not to enable security settings. For example, when testing with a host with a well-signed certificate, verify_ssl can be set to true. To see how we create the extension and its components, please see container_service_extension/mqtt_extension_manager.py and look for the create_* functions (for creating the extension, token, and api filter)